src/views/v2/rsvp/ari/form.php

```diff
@@ -13,12 +13,14 @@
  * @var Tribe__Tickets__Ticket_Object $rsvp The rsvp ticket object.
  * @var WP_Post|int $post_id The post object or ID.
  *
- * @since 4.12.3
```

Comment:
Instead of overwriting the `@since` we usually append a new one. Each `@since` should outline what was changed exactly like what you did.

Reply:
@redscar Thanks for pointing this out to me.